### PR TITLE
Add the release note for 1.29.1

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

File: site/src/pages/release-notes/1.29.1.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## 🛠️ Bug fixes
+
+- The <type://ExchangeType> is now correctly detected for the default <type://WebClient>. #5787
+- A duplicate key exception isn't raised anymore when building a JSON schema. #5788
+- The duplicator's child stream subscriber methods are now called by the correct executor. #5783
+
+## ☢️ Breaking changes
+
+- The signatures of <type://GrpcExceptionHandlerFunction#apply(RequestContext,Status,Throwable,Metadata)?full>
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.29.1.mdx:13]: Would you mind writing what problem is fixed by this change?

**Review by trustin — APPROVED** [review]

File: site/src/pages/release-notes/1.29.1.mdx (hunk 1 above)
Comment (minwoox) [reply site/src/pages/release-notes/1.29.1.mdx:15]: Thanks! updated. 😉

**Review by ikhoon — APPROVED** [review]
Thanks!

**Comment by minwoox** [comment]
screencapture-localhost-8000-release-notes-1-29-1-2024-06-28-11_14_26

**Comment by minwoox** [comment]
Thanks for the review. 😉

**Pull request merged**